### PR TITLE
PUBLISH_OSNICK, and adding circle variables

### PR DESCRIPTION
**Pull request by chayim**

### CODE REVIEW

File: build/docker/Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@ REDIS_VERSION=6.2.5
 PRODUCT=rebloom
```
Comment (rafie) [inline build/docker/Makefile:2]: We can add
```
ROOT=../..
include $(ROOT)/deps/readies/mk/main
```
at the top.

File: build/docker/Makefile [hunk 2]
```diff
@@ -2,7 +2,7 @@ REDIS_VERSION=6.2.5
...
 
 OSNICK ?= buster
-OSNICK_OFFICIAL ?= buster
+PUBLISH_OSNICK ?= buster
```
Comment (rafie) [inline build/docker/Makefile:5]: `buster` -> `bullseye`

File: .circleci/config.yml [hunk 3]
```diff
@@ -145,13 +145,13 @@ commands:
       - run:
           name: Build for platform (docker build)
           command: |
-            make -C build/docker build DOCKER_SUFFIX=".<<parameters.platform>>" OSNICK=<<parameters.platform>> TEST=1 VERBOSE=1
+            make -C build/docker build DOCKER_SUFFIX=".<<parameters.platform>>" OSNICK=<<parameters.platform>> CIRCLE_BRANCH=${CIRCLE_BRANCH} CIRCLE_TAG=${CIRCLE_TAG} TEST=1 VERBOSE=1
```
Comment (rafie) [inline .circleci/config.yml:148]: Don't we process `CIRCLE_BRANCH`/`CIRCLE_TAG` inside `mk/docker.rules`?
Also, do we still need an explicit `DOCKER_SUFFIX` here?

Comment (chayim) [reply .circleci/config.yml]: Correct - removed in the latest push.

File: build/docker/Makefile (hunk 2 above)
Comment (chayim) [reply build/docker/Makefile]: Updated in latest

**Pull request closed without merging**